docs/src/security/sso.md

```diff
@@ -131,184 +131,342 @@ lakeFS Cloud uses Auth0 for authentication and thus support the same identity pr
-    ```
\ No newline at end of file
+    **Helm Configuration (`values.yaml`):**
+
+    ```yaml
+    ingress:
+      enabled: true
+      ingressClassName: <class-name>
+      hosts:
+        - host: <lakefs.ingress.domain>
+          paths:
+            - /
+
+    lakefsConfig: |
+      auth:
+        external_aws_auth:
+          enabled: true
+          # the maximum age in seconds for the GetCallerIdentity request
+          #get_caller_identity_max_age: 60
+          # headers that must be present by the client when doing login request
+          required_headers:
+            # same host as the lakeFS server ingress
+            X-LakeFS-Server-ID: <lakefs.ingress.domain>
+    ```
+
+### Common Configuration
+
+All authentication methods share some common configuration options:
+
+```yaml
+# Enable enterprise features
+enterprise:
+  enabled: true
+
+# Common lakeFS configuration
+lakefsConfig: |
+  # Enable local RBAC for enterprise
+  features:
+    local_rbac: true
+  
+  # Basic auth configuration
+  auth:
+    encrypt:
+      secret_key: <your-encryption-secret>  # Set via secrets.authEncryptSecretKey
+    login_duration: 24h                      # Session duration
+    login_max_duration: 168h                 # Maximum session duration
+```
+
+### Secrets Management
+
+The Helm chart will automatically create Kubernetes secrets for sensitive values:
+
+```yaml
+secrets:
+  # Required: encryption key for auth cookies
+  authEncryptSecretKey: <your-random-secret-string>
+  
+  # For database connection (if using external DB)
+  databaseConnectionString: <postgres-connection-string>
+
+# Or use existing secrets
+secrets:
+  existingSecret: my-lakefs-secrets
+  # Define the keys in your secret:
+  authEncryptSecretKeyName: authEncryptSecretKey
+  databaseConnectionStringKeyName: databaseConnectionString
+```
+
+Authentication provider secrets are managed separately via the `enterprise.auth.*` configuration.
+
+### Environment Variables
+
+All configurations can also be set via environment variables:
+
+```bash
+# SAML
+LAKEFS_AUTH_PROVIDERS_SAML_SP_ROOT_URL=https://lakefs.company.com
+LAKEFS_AUTH_COOKIE_AUTH_VERIFICATION_EXTERNAL_USER_ID_CLAIM_NAME=samName
+
+# OIDC
+LAKEFS_AUTH_PROVIDERS_OIDC_CLIENT_ID=your-client-id
+LAKEFS_AUTH_OIDC_FRIENDLY_NAME_CLAIM_NAME=name
+
+# LDAP
+LAKEFS_AUTH_PROVIDERS_LDAP_SERVER_ENDPOINT=ldaps://ldap.company.com:636
+LAKEFS_AUTH_PROVIDERS_LDAP_BIND_DN=uid=bind-user,ou=Users,o=org,dc=company,dc=com
+
+# External AWS Auth
+LAKEFS_AUTH_EXTERNAL_AWS_AUTH_ENABLED=true
+```
+
+### Migration from Fluffy
+
+If you're upgrading from a version that used the Fluffy authentication service:
+
+1. Remove all `fluffy.*` configuration from your values.yaml
+2. Move authentication settings to the appropriate sections as shown above
+3. Update ingress rules to remove Fluffy-specific paths
+4. The chart will automatically detect and prevent deployment if Fluffy configuration is present
```

Comment:
The migration guide is missing specifics. 
1. minimum version (x.y.z ye ye) 
2. comparison of values, i.e old values.yaml vs new values.yaml. There is no documentation for all that. That's the only place! All fields.